src/hooks/useContent.js

```diff
@@ -30,23 +29,28 @@ import { getUserEditBranch, getUsersWorkingBranch } from "../core";
  *      - resourceStatus - is object containing details about problems fetching resource
  *      - error - Error object that has the specific error returned
  * @param {object} httpConfig - optional config settings for fetches (timeout, cache, etc.)
+ * @param {string} viewMode - list or markdown view
+ * @param {function} useUserLocalStorage
  */
 const useContent = ({
-  listRef,
-  contentRef,
-  verse,
+  listRef = 'master',
+  contentRef = 'master',
+  verse = 1,
   owner,
   server,
-  chapter,
-  filePath,
+  chapter= 1,
+  filePath = '',
   projectId,
   languageId,
   resourceId,
-  fetchMarkdown,
+  fetchMarkdown = true,
   onResourceError,
   httpConfig = {},
+  viewMode = 'markdown',
+  useUserLocalStorage,
 }) => {
   const [initialized, setInitialized] = useState(false)
+  const [processedItems, setProcessedItems] = useState(null)
```

Comment:
Should we move this line of code inside useExtraContent custom hook and return processedItems from it to avoid unnecessary re-renders? @PhotoNomad0

Reply:
🤦‍♂️Good point!

Reply:
Got missed when I split out to useExtraContent. @mannycolon

Reply:
No worries!